apps/dashboard/config/locales/en.yml

```diff
@@ -108,6 +108,8 @@ en:
     batch_connect_sessions_delete_title: "Delete"
     batch_connect_sessions_cancel_confirm: "Are you sure?"
     batch_connect_sessions_cancel_title: "Cancel"
+    batch_connect_sessions_relaunch_confirm: "Are you sure?"
```

Comment:
Surely we can reuse an `Are you sure?` from somewhere else and/or make a generic one.

Reply:
I forgot to remove this item. This was from the initial prototype.
The current implementation stars the session without prompting the user to confirm and shows the full page spinner.

Do you prefer the the button to show the `Are you sure?` confirmation message instead of the full page spinner?
Here is the current demo:
https://github.com/OSC/ondemand/issues/2522#issuecomment-1410408668

Reply:
Got it, that makes sense. No, they should just boot it up. I think we only need to confirm when something is destructive.

Reply:
fixed